packages/@react-aria/interactions/src/Pressable.tsx

```diff
@@ -19,6 +19,8 @@ interface PressableProps extends PressProps {
 }
 
 export const Pressable = React.forwardRef(({children, ...props}: PressableProps, ref: RefObject<HTMLElement>) => {
+  // FIXME ???
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   ref = ref || useRef();
```

Comment:
Is this actually allowed?

Reply:
Nope
Should be able to put it on its own line right before and then 'or'